docs/whats_new/version_0_24_0.rst

```diff
@@ -0,0 +1,21 @@
+Version 0.24.0
+--------------
+
+Search size for rtt segger
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Added the possibility to search in a given range for the block of the device
+starting from the specified block address.
+
+Tags in global-config
+^^^^^^^^^^^^^^^^^^^^^
+
+Added tags to global config.
```

Comment:
Is there any documentation regarding on how to use it?

Reply:
The Documentation already exists in advanced_usage/access_config. It does not change anything but rather provide more information